js/jsonld.js

```diff
@@ -1646,6 +1646,30 @@ jsonld.cache = {
 };
 
 /**
+ * Accept header.
+ */
+jsonld.acceptHeader = 'application/ld+json, application/json';
+
+/**
+ * Build an headers object from custom headers and assert `accept` header isn't overridden.
+ *
+ * @param {Object} headers an object of headers with key as header name and value as header value.
+ * @return {Object} an object of headers with a valid `accept` header.
+ */
+jsonld.buildHeaders = function (headers) {
+  headers = headers || {};
+  if ('Accept' in headers || 'accept' in headers) {
```

Comment:
Nits: Remove space after `if` and `function` to keep in the existing code style of this module.

Also, this check here will cover only two possible capitalizations of `accept`. It's annoying, but perhaps we should do this instead:

```js
var hasAccept = Object.keys(headers).map(function(h) {
  return h.toLowerCase();
}).indexOf('accept') !== -1;
if(hasAccept) {
  // throw range error
}
```